openapi.yaml

```diff
@@ -35,6 +35,12 @@ paths:
   /projects/{platform}/{org}/{repo}/badge:
     get:
       parameters:
+        - in: query
+          name: style
+          type: string
+          required: false
+          default: flat
+          description: Badge style (plastic, flat, flat-square, for-the-badge and social)
```

Comment:
Add a `pattern` field which would only allow these specific styles as input to this.